domain/Grant.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+class Grant {
```

Comment:
I know this is self-explanatory, but you could add a brief comment to describe what the class is for.